src/service/sdkManager.js

```diff
@@ -151,10 +161,50 @@ const updateInactiveBroadcastState = (event) => {
 }
 
 const updateLayersBroadcastState = (event) => {
-  if ('0' in event.data.medias) 
+  if ('0' in event.data.medias) {
     layers.updateLayers(event)
-  else
+  } else {
     layers.deleteLayers()
+  }
+  const medias = state.Layers.mainTransceiverMedias.active
+  if (medias.length === 0) {
+    console.warn('No active layers available, will wait for next event. Switching to Auto until then.')
+    selectingLayerTimeout != null && clearTimeout(selectingLayerTimeout)
```

Comment:
I'm wondering, why did you use this syntax and not an if?

Reply:
You are right, I will change it